e2e/test/iothub/messaging/MessageFeedbackE2ETests.cs

```diff
@@ -1,82 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Threading;
-using System.Threading.Tasks;
-using FluentAssertions;
-using Microsoft.Azure.Devices.Client;
-using Microsoft.Azure.Devices.E2ETests.Helpers;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-
-namespace Microsoft.Azure.Devices.E2ETests.Messaging
-{
-    [TestClass]
-    [TestCategory("E2E")]
-    [TestCategory("IoTHub")]
-    public class MessageFeedbackE2ETests : E2EMsTestBase
```

Comment:
This test class doesn't cover anything interesting anymore now that the receiveAsync call is gone and the callbacks are executed in order. We have a separate test class for the actual message feedback receiving part of this test